task/01-strings-tasks.js

```diff
@@ -22,7 +22,10 @@
  *   '',  'bb'  => 'bb'
  */
 function concatenateStrings(value1, value2) {
-    throw new Error('Not implemented');
+
+    return value1+value2;
```

Comment:
It is necessary to respect the indentation between operators
https://javascript.info/coding-style